packages/webamp/js/components/FFTNullsoft.ts

```diff
@@ -0,0 +1,206 @@
+// The Web Audio API's FFT is bad, so this exists now!
+// Taken from https://github.com/WACUP/vis_classic/tree/master/FFTNullsoft
+
+export class FFT {
+  private bitrevtable: number[];
+  private envelope: Float32Array | null;
+  private equalize: Float32Array | null;
```

Comment:
See below comments. I think these don't need to be nullable?
```suggestion
  private envelope: Float32Array;
  private equalize: Float32Array;
```